prow/config/presubmits.yaml

```diff
@@ -734,6 +734,172 @@ presubmits:
           type: DirectoryOrCreate
 
   maistra/istio:
+  - name: istio_2.1-1.9-unit
+    trigger: (?m)^/test( | .* )unit,?($|\s.*)
+    decorate: true
+    always_run: true
+    path_alias: istio.io/istio
+    skip_report: false
+    max_concurrency: 2
+    branches:
+      - maistra-2.1-istio-1.9
```

Comment:
I think other jobs would have to be changed to the more strict `^maistra-2.1$` (add changing this one here also doesn't hurt) otherwise pr's in this branch would trigger jobs in the maistra-2.1 branch.

Another solution would be to use maistra-2.2 as the branch name. /me runs

Reply:
good catch, thank you!